### ISSUE TITLE
Any interest in porting this to Sublime Text 4

### ISSUE THREAD

**Issue by andregoldstein** [ahmadawais/shades-of-purple-vscode#109]
Hi there, I really love using your theme in VS Code. I was just seeing if there was any interest or possibility in porting it to Sublime Text?

As much as I love VS Code when I want a superfast editor ST4 is just orders or magnitude faster. They've also just released a version for the Apple M1 chip.

Thanks and cheers in either case.

**Comment by dwhenson** [comment]
Yes please this would be so nice!

**Comment by ahmadawais** [comment]
Of course I am. Spent ten years with Sublime before switching to VSCode (and publishing https://VSCode.pro course). But want to build a Sublime theme. If you can help that'd be great.

**Comment by jbrooksuk** [comment]
I converted the basic colour scheme using `code2subl`. https://gist.github.com/jbrooksuk/8d758300b0e028e9130cebee6d494ee2

**Comment by t-lock** [comment]
YES! I just came here to make an issue. My only issue is its not on ST4! :)

**Comment by ahmadawais** [comment]
Need to buy ST4 to code/test one.

**Comment by kapitanluffy** [comment]
Thank you for this awesome theme @ahmadawais! Just so you know, you don't need to buy ST4.
